src/DataGrid.tsx

```diff
@@ -265,15 +264,13 @@ function DataGrid<R, K extends keyof R, SR>({
               sortColumn={props.sortColumn}
               sortDirection={props.sortDirection}
               onSort={props.onSort}
-              scrollLeft={nonStickyScrollLeft}
             />
             {enableFilters && (
               <FilterRow<R, SR>
                 height={headerFiltersHeight}
                 width={columnMetrics.totalColumnWidth + getScrollbarSize()}
                 lastFrozenColumnIndex={columnMetrics.lastFrozenColumnIndex}
                 columns={viewportColumns}
-                scrollLeft={nonStickyScrollLeft}
```

Comment:
Do you think we can remove `scrollLeft` from `InteractionMasks` also?

Reply:
I believe we need it to position the editor, and close the editor when scrolling.
We can look into it further in the future.